src/components/CustomSelectDropdown/index.tsx

```diff
@@ -36,6 +36,7 @@ const CustomSelectDropdown = ({
         onClick={() => setToggleDropdown(!is_toggle_dropdown)}
         onKeyDown={(e) => e.key === 'ArrowDown' && setToggleDropdown(!is_toggle_dropdown)}
         className={`${styles.selectWrapper} ${toggle_dropdown} ${is_error ? styles.error : ''}`}
+        data-testid='custom-dropdown'
```

Comment:
`data-testid` shouldn't be hard coded value

Reply:
Could please follow our naming style for test id as well

Reply:
Done, applied both of your feedback